tracetools_launch/tracetools_launch/action.py

```diff
@@ -445,7 +469,8 @@ def setup() -> bool:
                 )
                 if self._trace_directory is None:
                     return False
-                self._logger.info(f'Writing tracing session to: {self._trace_directory}')
+                if not dual_session:
+                    self._logger.info(f'Writing tracing session to: {self._trace_directory}')
```

Comment:
We should still print the trace directory if it's a dual session.

In the case of a dual session (snapshot session) here, the trace directory path returned by `lttng.lttng_init()` should be `~/.ros/tracing/$session_name/snapshot`. I think it's fine to print that here.

```suggestion
                self._logger.info(f'Writing tracing session to: {self._trace_directory}')
```

Reply:
I think we should say some thing like "Tracing session will be written to:" ?

Reply:
See https://github.com/ros2/ros2_tracing/pull/191#discussion_r2291567646